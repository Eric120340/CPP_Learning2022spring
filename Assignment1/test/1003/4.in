7 4
80 40 30 41 
69 46 60 12 
28 16 76 45 
84 29 99 73 
17 65 1 43 
10 54 43 91 
76 67 91 87 
4 1
73 
94 
18 
52 
