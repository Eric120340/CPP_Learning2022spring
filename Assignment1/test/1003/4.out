12272 
11065 
7256 
14436 
9605 
11312 
18008 
